### PR TITLE
Add sort, order and offset to client.data_object.get()

### PR DESCRIPTION
**Pull request by Sasmuin**
Close https://github.com/semi-technologies/weaviate-python-client/issues/188.
Adds the missing parameters sort, order and offset to the function client.data_object.get().

### CODE REVIEW

File: weaviate/data/crud_data.py [hunk 1]
```diff
@@ -438,6 +438,9 @@ def get(
         node_name: Optional[str] = None,
         consistency_level: Optional[ConsistencyLevel] = None,
         limit: Optional[int] = None,
+        offset: Optional[int] = None,
+        sort: Union[str, List[str], None] = None,
+        ascending: Union[bool, List[bool]] = True,
```
[the remark this answers is not in the document]
Comment (dirkkul) [reply weaviate/data/crud_data.py:443]: I think "order", or "sorting_order" would be better names.

Comment (dirkkul) [inline weaviate/data/crud_data.py:443]: I'm not convinced that having two different parameters to control one functionality is the best option and might be confusing for our users. Why do you think having two is better than one?

**Review by dirkkul — CHANGES_REQUESTED** [review]
Looks nice, thank you for your contribution! Some smaller changes are needed.

**Comment by Sasmuin** [comment]
I tried making the requested changes.

File: weaviate/data/crud_data.py (hunk 1 above)
Comment (Sasmuin) [reply weaviate/data/crud_data.py]: The current implementation uses a list for sort, where sort[0] are the properties and sort[1] the order. Do you think using a dictionary would be better?

Comment (StefanBogdan) [reply weaviate/data/crud_data.py]: With a `dict` the order does not matter and the `"order"` can always be omitted to have the default one (which is `asc`).

**Comment by Sasmuin** [comment]
I changed sort to a dictionary now. I did however rename 'order' to 'order_asc' to make it clear that booleans should be used to give the order of the properties.

**Review by dirkkul — APPROVED** [review]

**Comment by Sasmuin** [comment]
I agree with the CLA.

**Review by StefanBogdan — APPROVED** [review]
Looks good!

**Pull request merged**